apps/ejabberd/src/ejabberd_auth_jwt.erl

```diff
@@ -213,3 +228,11 @@ get_auth_opt(Host, Key) ->
                     undefined
             end
     end.
+
+set_auth_opts(Host, KVs) ->
```

Comment:
Merging logic in the function description.
Like what would happen with old options?